### ISSUE TITLE
To have the ability to use SSL without verifying certificates

### ISSUE THREAD

**Issue by saurav2222** [bulkan/robotframework-requests#120]
I am using Create session to a url which does not have a valid ssl certificate. So i am using something like below

Create Session  test  "URL"  verify=False
${resp}= Get Request test  "URL"
Should be equal As Strings  ${resp.status_code}  200
Dictionary should contain value  ${resp.json()}   "some text"

when i run this test i get an error 
KEYWORD RequestsLibrary . Create Session MD1, https://dms.md1.drivescale.com, verify=False
KEYWORD ${resp} = RequestsLibrary . Get Request MD1, /ds/user?properties=username
Documentation: 	

Send a GET request on the session object found using the
Start / End / Elapsed: 	20160629 00:02:08.563 / 20160629 00:02:09.246 / 00:00:00.683
00:02:08.602 	INFO 	Starting new HTTPS connection (1): dms.md1.drivescale.com 	
00:02:09.246 	FAIL 	SSLError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:590)

The url i am trying to use is a developers build so it does not have a valid certificate.
this url cannot be accessed outside our work network.
Please have a look into this and provide a fix or a workaround.

**Comment by bulkan** [comment]
Hi there which version of `robotframework-requests` are you using ?

**Comment by vkosuri** [comment]
@saurav2222 could you try like this?

``` robot
Create Session alias=test url=URL verify=${False}
${resp}= Get Request test "URL"
Should be equal As Strings ${resp.status_code} 200
```

**Comment by saurav2222** [comment]
@vkosuri i get the same results when i use verify=${False} 
But anyways i have the certificate for the URL now and it works 
I have removed verify now and i believe verify=False is set by default
I believe the issue was due to the authentication so now when i have provided authentication it works fine.
But i think when we specify verify=${False} it does not work properly because in my case with auth it works fine but with verify=${False} it fails with the same error as above

**Comment by vkosuri** [comment]
@saurav2222 I have added a testcase with verify False, it was passed successfully, I think the problem could be with your credentials..

```robot
Get HTTPS & Verify Cert False
    [Tags]  get
    Create Session    httpbin    https://httpbin.org   verify=False

    ${resp}=  Get Request  httpbin  /get
    Should Be Equal As Strings  ${resp.status_code}  200
```

**Comment by saurav2222** [comment]
@vkosuri not any issue rather a question related to the Requests library
    Create session  MD1  https://dms.md1.drivescale.com  auth=${cred}
    ${resp}= 	Get request 	MD1     /ds/user?properties=username%2Cid
Now ${resp.json()} has the following data
[{
	u 'username': u 'sk3@ds.com',
	u 'id': u '08f560ec-2382-4017-94fa-507c50d705c5'
}, {
	u 'username': u 'admin@drivescale.com',
	u 'id': u '0f7f1635-844d-4c09-a4b0-905e15e74238'
}, {
	u 'username': u 'ann.mccown@drivescale.com',
	u 'id': u '224f8b7e-2dc7-4875-a99e-9aacaff6377b'
}, {
	u 'username': u 'qa@drivescale.com',
	u 'id': u '5ca6072d-2130-4e51-9325-d020ab4c3e3b'
}, {
	u 'username': u 'dms-5910e279-9788-4121-a28c-688c6e8b5dc9@u59.r1.hq.drivescale.com',
	u 'id': u '78b42885-fa91-42ea-b523-756978b09a1f'
}, {
	u 'username': u 'myviewonly@ds.com',
	u 'id': u 'b0dcd7b9-fd1c-4a4b-8f1a-31cdc6fa824c'
}, {
	u 'username': u 'sk6@ds.com',
	u 'id': u 'b43b2568-9ffe-46ec-a7d4-ed0e2036ed28'
}, {
	u 'username': u 'test@drivescale.com',
	u 'id': u 'bbc765a1-d20f-4ac2-8e51-aaec381fa073'
}, {
	u 'username': u 'dms-72540eff-57ed-5d9e-a8ed-ffcb30e8d28d@u60.r1.hq.drivescale.com',
	u 'id': u 'd524e0a5-07b8-486e-b23e-b36c76276514'
}, {
	u 'username': u 'sk2@ds.com',
	u 'id': u 'db21a69d-4fa4-4e2f-83d1-113d073ed8be'
}, {
	u 'username': u 'sk5@ds.com',
	u 'id': u 'e30bdeba-5fae-4994-b3b9-73fd52a8c77a'
}, {
	u 'username': u 'clustermanager@ds.com',
	u 'id': u 'edbdc03b-1b8b-4bc6-8b97-f7a578a8c603'
}, {
	u 'username': u 'dms-05b9877c-fb06-421a-afb4-bb7d75fc51dc@u61.r1.hq.drivescale.com',
	u 'id': u 'f4c75046-986b-416c-92a8-93c335d39479'
}]

Now i need the id of the user "admin@drivescale.com" [any user]
basically i have the username and i want to get the id of that username
how can i possibly get this
I am new to this so just asking a question

Thanks for your help

**Comment by Rue1983** [comment]
Is the option verify=False supported in latest version? There was an error said:  "IOError: Could not find a suitable TLS CA certificate bundle, invalid path: False".

**Comment by karimbou8** [comment]
> @vkosuri i get the same results when i use verify=${False}
> But anyways i have the certificate for the URL now and it works
> I have removed verify now and i believe verify=False is set by default
> I believe the issue was due to the authentication so now when i have provided authentication it works fine.
> But i think when we specify verify=${False} it does not work properly because in my case with auth it works fine but with verify=${False} it fails with the same error as above
> @bulkan I am using 0.4.5

Hello
please how did you found solution ?

**Comment by lucagiove** [comment]
This issue is closed.
verify=${False} is available since long time consider updating the library or ask a more precise scenario.
